maps/Aurora-MapDev/aurora-6.dmm

```diff
@@ -273,6 +273,8 @@
 "fm" = (/obj/structure/cable/blue{d1 = 1;d2 = 2;icon_state = "1-2"},/turf/simulated/floor/tiled,/area/tcommsat/lounge)
 "fn" = (/obj/structure/cable/blue{d1 = 1;d2 = 2;icon_state = "1-2"},/turf/simulated/floor/tiled,/area/tcommsat/lounge)
 "fo" = (/obj/machinery/atmospherics/pipe/simple/hidden/supply{dir = 4},/obj/machinery/atmospherics/pipe/simple/hidden/scrubbers{dir = 4},/obj/structure/cable/blue{d1 = 4;d2 = 8;icon_state = "4-8"},/obj/structure/cable/blue{d1 = 1;d2 = 8;icon_state = "1-8"},/turf/simulated/floor/tiled,/area/tcommsat/lounge)
+"fp" = (/obj/effect/landmark/corpse/engineer/rotten,/turf/simulated/floor/asteroid,/area/mine/explored)

```

Comment:
It was always meant to be there. I'm not quite sure what happened to it. Reminds me that I need to re-re-remap Poly, however.